### ISSUE TITLE
Error in class TiffOutput

### ISSUE THREAD

**Issue by StephanAllgeier** [modsim/CellSium#2]
I get an error when writing the output to tiff files. This happens both in a Windows and in an Ubuntu environment, both using conda.

My command line input is `python -m cellsium -t SimulationDuration=8 --Output TiffOutput -o ./sim1 -t Seed=1`

The error output (in Visual Studio Code) after the simulation is:
```
Exception ignored in: <function TiffOutput.__del__ at 0x000002C64C30B7E0>
Traceback (most recent call last):
  File "C:\Users\qj8035\.conda\envs\cellsium\Lib\site-packages\cellsium\output\render.py", line 740, in __del__
    binary_rois = [ImagejRoi.frompoints(**roi).tobytes() for roi in self.rois]
  File "C:\Users\qj8035\.conda\envs\cellsium\Lib\site-packages\cellsium\output\render.py", line 740, in <listcomp>
    binary_rois = [ImagejRoi.frompoints(**roi).tobytes() for roi in self.rois]
TypeError: ImagejRoi.frompoints() got some positional-only arguments passed as keyword arguments: 'points'
```

The error happens in render.py in line 740:
https://github.com/modsim/CellSium/blob/208e115ff9acf8f29b1fcaa69434920aefd2ecc7/cellsium/output/render.py#L740

For me, the following change fixed the issue (the `"points"`-values from the `roi` dictionary can't be passed by name):
```
binary_rois = [ImagejRoi.frompoints(roi["points"], position=roi["position"], index=roi["index"], t=roi["t"]).tobytes() for roi in self.rois]
```

**Comment by dpinte** [comment]
For the record, it's a dependency issue. The current version of CellsSium seems to require the `roifile` version to be equal to `2021.6.6`

**Comment by JojoDevel** [comment]
Thanks for getting in touch and solving the error @StephanAllgeier and @dpinte. I merged the code changes #3
